ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerGeoprocessing/LocalServerGeoprocessing.qml

```diff
@@ -82,6 +82,7 @@ LocalServerGeoprocessingSample {
                             verticalCenter: parent.verticalCenter
                             margins: 5
                         }
+                        width: 100
```

Comment:
is this change intended for this PR?

Reply:
Actually yes - when testing the local server sample the input text box was running over the label text. This resolves that UI bug.